js/app/src/Index.svelte

```diff
@@ -210,9 +210,8 @@
 			status: "running",
 			detail: "RUNNING"
 		};
+		console.log("Index wrapper: ", wrapper);
 
-		await mount_custom_css(wrapper, config.css);
-		css_ready = true;
```

Comment:
What is this?

Reply:
this came from your pr here: https://github.com/gradio-app/gradio/pull/4572/files#diff-5ca8aa298aaa3f175747debafd686b0270764ffc62793997deff716496c27a75 so will leave it out